requirements/build_windows.txt

```diff
@@ -1,8 +1,10 @@
 -r common.txt
 flake8==3.7.7
 isort==4.3.20
-marshmallow==2.18.1
-pytest==4.6.3
 wheel==0.33.4
-pytest-xdist==1.29.0
 numpy==1.15.4
+
```

Comment:
Any reason for the reordering and blank lines? Just curious

Reply:
Mostly as a visual aid as I was comparing requirements across files. I do like to have requirements having a similar purpose grouped together though, like the pytest requirements. I can remove the blank lines, or completely revert the file if you'd prefer.